extensions/wikia/Flags/scripts/FlagsModal.js

```diff
@@ -1,38 +1,50 @@
 require(['jquery', 'wikia.loader', 'mw'], function ($, loader, mw) {
 	'use strict';
 
+	/* Modal buttons config for done and cancel buttons */
+	var buttonsForFlagsExistingState = [{
+		vars: {
+			value: mw.message('flags-edit-modal-done-button-text').escaped(),
+			data: [
+				{
+					key: 'event',
+					value: 'done'
+				}
+			]
+		}
+	},
+	{
+		vars: {
+			value: mw.message('flags-edit-modal-cancel-button-text').escaped(),
+			data: [
+				{
+					key: 'event',
+					value: 'close'
+				}
+			]
+		}
+	}],
+
+	/* Modal close button config*/
+	buttonForEmptyState = [{
+		vars: {
+			value: mw.message('flags-edit-modal-close-button-text').escaped(),
+			data: [
+				{
+					key: 'event',
+					value: 'close'
+				}]
```

Comment:
I'm not sure if our guidelines touch it but you can make it consistent in the file and move `]` to the next line just like above :japanese_ogre: